_data/projects/GreenBerry.yml

```diff
@@ -0,0 +1,10 @@
+name: GreenBerry
+desc: one-line statement language
+site: https://abdur-rahmaanj.github.io/greenBerry/
+tags:
+- python
+- new-language
+- cli
+upforgrabs:
+  name: up-for-grabs
```

Comment:
@Abdur-rahmaanJ This is the issue label name that you are using to create issues for contributors.